src/evaluate.cpp

```diff
@@ -23,26 +23,33 @@
 #include <iomanip>
 #include <sstream>
 #include <iostream>
+#include <vector>
 
 #include "bitboard.h"
 #include "evaluate.h"
 #include "material.h"
+#include "misc.h"
 #include "pawns.h"
 #include "thread.h"
 #include "uci.h"
 
 namespace Eval {
 
   bool useNNUE;
-  std::string eval_file_loaded="None";
+  std::string eval_file_loaded = "None";
 
   void init_NNUE() {
 
+    std::vector<std::string> dirs = { "" , CommandLine::binaryDirectory };
```

Comment:
what about adding something as in https://github.com/official-stockfish/Stockfish/issues/3030#issuecomment-679107572 ?

Reply:
I am working on it

Reply:
See https://github.com/official-stockfish/Stockfish/pull/3070